packages/core/src/extensions/renderer-api/components.ts

```diff
@@ -36,7 +36,6 @@ export * from "../../renderer/components/layout/wizard-layout";
 export * from "../../renderer/components/layout/tab-layout";
 
 // form-controls
-export * from "../../renderer/components/button";
```

Comment:
You need to replace this export with one from `@k8slens/button` to maintain backwards compatibility

Reply:
check